packages/astro/test/streaming.test.js

```diff
@@ -33,11 +35,21 @@ describe('Streaming', () => {
 			let res = await fixture.fetch('/');
 			let chunks = [];
 			for await (const bytes of streamAsyncIterator(res.body)) {
-				let chunk = bytes.toString('utf-8');
+				let chunk = decoder.decode(bytes);
 				chunks.push(chunk);
 			}
 			expect(chunks.length).to.be.greaterThan(1);
 		});
+
+		it('Body of slots is chunked', async () => {
+			let res = await fixture.fetch('/slot');
+			let chunks = [];
+			for await (const bytes of streamAsyncIterator(res.body)) {
+				let chunk = decoder.decode(bytes);
+				chunks.push(chunk);
+			}
+			expect(chunks.length).to.be.greaterThan(2);
```

Comment:
Is `2` exported the number of slots in the test fixture? Is it because we render two `Wait` components?

Reply:
3 is the expected. You get the first initial chunk. Then the chunk after the first wait. Then the chunk after the second wait. I guess I could change this to equal because it should always be 3.

Reply:
It might help. Having a strict equal can prevent possible regressions

Reply:
The reason for avoiding it is that Node buffers writes. If you send 2 chunks with no wait it will combine them into a single write on the network. So the chunks can be less than the number you expect, but never more. In this case, the wait is sufficiently long enough that it should always split into 3 chunks. SO I've updated the test to reflect that.